docs/section5/Rule5-19.md

```diff
@@ -18,26 +18,34 @@
 **Data Lookup:** Table G3.1.1-1  
 **Function Call:**  
 
-  1. get_overall_building_segment_wwr()  
+  1. get_building_segment_window_wall_areas()  
   2. data_lookup()  
```

Comment:
I didn's see function under that name. Did you mean "get_window_wall_areas" function?